protos/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ts-proto-descriptors",
-  "version": "1.6.0",
```

Comment:
@stephenh The deployment fixes the build errors in the root project. No other changes needed.

To prove this, with this PR checked out,

 1. Cd to "protos" and run "yarn link"
 2. Cd to root and run "yarn link ts-proto-descriptors"
 3. Now all of the tests / build commands pass.

Otherwise there are errors related to type mismatches between the generated plugin messages.

Reply:
Okay, @paralin I've published ts-proto-descriptors 1.7.0 in this PR:

https://github.com/stephenh/ts-proto/pull/606

(Which I merged, although ts-proto-descriptors is still built by hand.)

I ran the `yarn link` as you suggested, but had to bump TypeScript in the `protos/package.json` itself, and also set strict: false, which kinda sucks but I think is a separate issue to fix in the `unknownFields` option.

Also, I noticed for the 1st time that the `Exact` type gets erased/inline in the `*.d.ts` files:

```
    fromPartial<I extends {
        path?: number[];
        sourceFile?: string;
        begin?: number;
        end?: number;
    } & {
        path?: number[] & number[] & Record<Exclude<keyof I["path"], keyof number[]>, never>;
        sourceFile?: string;
        begin?: number;
        end?: number;
    } & Record<Exclude<keyof I, keyof GeneratedCodeInfo_Annotation>, never>>(object: I): GeneratedCodeInfo_Annotation;
```

Which is weird! I didn't know that would happen/was happening... Have you seen that before? I was trying to google/read the docs on it, but I'm not sure what it's called.

The "inlined" `Exact` type was the source of one of the errors in `schema.ts` that was fixed by bumping `protos/package.json`s version TypeScript of...no idea why...

But, all that said, does ts-proto-descriptors 1.7.0 work for you/this PR?

Reply:
Updated the PR, if tests pass, then yup should be all good!